src/common/libutil/base256.h

```diff
@@ -0,0 +1,52 @@
+/************************************************************\
+ * Copyright 2016 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef UTIL_BASE256_H
+#define UTIL_BASE256_H
+
+#include <stdbool.h>
+
+#define BASE256_ENCODED_SIZE(x) ((x * 4) + 4 + 1)
+#define BASE256_DECODED_SIZE(x) ((x / 4) - 4 - 1)
+#define BASE64_PREFIX "🇫"
+#define is_base256(x) (strlen((x)) > 4 && memcmp ((x), BASE64_PREFIX, 4) == 0)
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*  Decode string `data` in base256 into result `buf` of size `buflen`,
+ *  which should be at least size BASE256_DECODED_SIZE(strlen(in))
+ *
+ *  Returns 0 on success, -1 on error with errno set:
+ *  EINVAL: invalid arguments
+ *  ENOENT: an invalid character appears in data
+ */
+extern int base256_decode (void *buf,
```

Comment:
AFAIK `extern` has no effect here and probably should be dropped for consistency wtih the rest of flux.

Reply:
Oops, sorry, I probably missed some things when "converting" the original code ...